src/bitmaskops.cu

```diff
@@ -0,0 +1,100 @@
+#include <gdf/gdf.h>
+#include <gdf/utils.h>
+#include <gdf/errorutils.h>
+#include <gdf/cffi/functions.h>
```

Comment:
`gdf/cffi/functions.h` shouldn't be referenced directly and is already included by `gdf/gdf.h`